cablelabs/lpwanserver
# 1.  All protocol handlers register their protocol on startup.  there …

…is no need to add protocols anymore.
2.  Return meta data with protocols including customization fields.
3. Added upsert function to INetworkProtocols and such to support registering protocols every time.

#### What does this PR do?
1.  All protocol handlers register their protocol on startup.  there is no need to add protocols anymore.
2.  Return meta data with protocols including customization fields.
3. Added upsert function to INetworkProtocols and such to support registering protocols every time.
#### Do you have any concerns with this PR?
No its big but straightforward
#### How can the reviewer verify this PR?
Run it with an empty database and GET /api/networkProtocols and you should see 4 with metadata (TMobileNBIoT is there to support testing.  Don't use it)
#### Any background context you want to provide?
#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves?
Yes
- Does the documentation need an update?
Yes
- Does this add new dependencies?
No
- Have you added unit or functional tests for this PR?
I reused existing.
